dapps/appkit-siwe/react/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "appkit-siwe-react",
+  "private": true,
+  "version": "0.0.1",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
```

```suggestion
    "dev": "vite --port 5174",
```

Hardcode port to match server CORS